src/index.html

```diff
@@ -6,9 +6,69 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet"
+      href="./style.css"
+    >
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+
+      <a href="#" class="header__logo">
+
+        <img src="./images/logo.png" alt="moyo image"
+            class="header__logo-img"
+        >
+
+      </a>
```

Comment:
```suggestion
    <header class="header">
      <a href="#" class="header__logo">
        <img
          src="./images/logo.png" 
          alt="moyo image"
          class="header__logo-img"
        >
      </a>
```
Pay attention to the formatting